modules/vaos/app/controllers/vaos/limits_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_dependency 'vaos/application_controller'
```

Comment:
Would prefer if these were removed by doing something like VAOSBaseController or something similar.

Reply:
To be fair, I think most (if not all) of the modules are doing something similar and we drew heavily from those examples, but is something I've addressed in a PR you've since approved so TY. :)